### PR TITLE
New version: Erlang.ErlangOTP version OTP-25.0-rc1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46566)

### CODE REVIEW

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Erlang.ErlangOTP
+PackageVersion: OTP-25.0-rc1
+PackageLocale: en-US
+Publisher: Erlang
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.locale.en-US.yaml:7]:
```suggestion
Publisher: Ericsson AB
```

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Erlang.ErlangOTP
+PackageVersion: OTP-25.0-rc1
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 25.0
```

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Erlang.ErlangOTP
+PackageVersion: OTP-25.0-rc1
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml:5]:
```suggestion
PackageVersion: 25.0
```

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Erlang.ErlangOTP
+PackageVersion: OTP-25.0-rc1
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.yaml:5]:
```suggestion
PackageVersion: 25.0
```

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+FileExtensions:
+- beam
+- erl
+- hrl
+ReleaseDate: 2022-02-16
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/erlang/otp/releases/download/OTP-25.0-rc1/otp_win32_25.0-rc1.exe
+  InstallerSha256: 042892A0F147D9B08E19079DB99233D7CA9C8E371B6D4DC034371AEB86B833EA
+  ProductCode: Erlang OTP 24 (12.2.1)
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml:20]:
```suggestion
  ProductCode: Erlang OTP
```

File: manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/erlang/otp/releases/download/OTP-25.0-rc1/otp_win64_25.0-rc1.exe
+  InstallerSha256: C8CD8A249BC5E9D37C064CB005B6709C872A2C2962C4084286B2E5225C4F2C4F
+  ProductCode: Erlang OTP 24 (12.2.1)
```
Comment (ItzLevvie) [inline manifests/e/Erlang/ErlangOTP/OTP-25.0-rc1/Erlang.ErlangOTP.installer.yaml:24]:
```suggestion
  ProductCode: Erlang OTP
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Eyyy, they finally released my changes

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
@vedantmgoyal2009 - May want to update Version Method for this one

**Pull request merged**